app/Providers/AppServiceProvider.php

```diff
@@ -25,11 +27,14 @@ public function boot()
      */
     public function register()
     {
-        //
         $this->app->singleton(PasswordStore::class, function ($app) {
             $passwordStoreManager = new PasswordStoreManager($app);
 
             return $passwordStoreManager->driver();
         });
+
+        $this->app->singleton(FakerGenerator::class, function () {
```

Comment:
cause with out it you get American addresses and phone numbers, this overriders the doctrine US default